connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -170,6 +170,11 @@ protected void close() {
         } catch (Throwable t) {
             log.warn("Could not close transformation chain", t);
         }
+        try {
+            retryWithToleranceOperator.close();
+        } catch (Throwable t) {
+            log.warn("Could not close retry operator", t);
+        }
```

Comment:
This is common enough that there's a util for that and is used extensively:

```suggestion
        Utils.closeQuietly(retryWithToleranceOperator, "retry operator");
```